src/main/java/org/omnifaces/util/Utils.java

```diff
@@ -1239,12 +1224,7 @@ public static String decodeURL(String string) {
             return null;
         }
 
-        try {
-            return URLDecoder.decode(string, UTF_8.name());
-        }
-        catch (UnsupportedEncodingException e) {
-            throw new UnsupportedOperationException(ERROR_UNSUPPORTED_ENCODING, e);
```

Comment:
Won't this leave behind an unused `ERROR_UNSUPPORTED_ENCODING` constant? If so then it should also be removed.